src/store/state.ts

```diff
@@ -27,8 +27,8 @@ const state: AppState = {
     {
       type: 'text',
       id: 'name',
-      label: 'Name',
-      description: 'Optional name',
+      label: 'Label',
```

Comment:
different issue , would be nice if we could use configurable 'i18n' database label

Reply:
we have those, all attributes in an entity have language-specific labels

Reply:
Nice but outside of the scope of this PR

I added an enhancement to the list of issues
https://github.com/molgenis/molgenis-app-lifelines-webshop/issues/177